metadata-ingestion/src/datahub/ingestion/transformer/base_transformer.py

```diff
@@ -17,13 +17,22 @@
 log = logging.getLogger(__name__)
 
 
-class LegacyMCETransformer(Transformer, metaclass=ABCMeta):
+class HandleEndOfStreamTransformer:
+    def handle_end_of_stream(
+        self, entity_urn: str
```

Comment:
this method should not take in an `entity_urn`. It should only be called once after all records have been processed, not once per urn

Reply:
done